packages/gatsby-theme-carbon/src/components/ImageGallery/ImageGalleryImage.module.scss

```
@@ -0,0 +1,33 @@
+.image-button-wrapper {
+  border: 0;
+  background: none;
+  padding: 0;
+  margin-bottom: $spacing-06;
+  outline: none;
+}
+
+.image-button-wrapper:focus img {
+  outline: 2px solid $focus;
+  outline-offset: -2px;
+}
+
+.image-title {
+  position: absolute;
+  display: block;
+  width: 100%;
+  align-self: flex-start;
+  left: $spacing-07;
+  color: $text-04;
+  font-weight: bold;
```

```suggestion
  @include carbon--type-style('productice-heading-02');
```

instead of just font-weight: bold